components/basecomponents/MemoryGame/Themes/MemoryGameTaleTheme.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styles from './MemoryGameTaleTheme.module.css';
+import MemoryGame from '../MemoryGameCore';
+import { TranslationJSON } from 'utils/Phrase';
+import { CountryVariant, getCountryVariant } from 'utils/locales';
+import kidsWords_CS from 'data/translations/cs/pro-deti.json';
+import kidsWords_SK from 'data/translations/sk/pro-deti_sk.json';
+import kidsWords_PL from 'data/translations/pl/pro-deti_pl.json';
+
+// const KIDS_WORDS: Record<CountryVariant, KidsTranlsation[]> = {
+//     cs: { /* filter tale words from kids json */ },
+//     sk: { /* filter tale words from kids json */ },
+//     pl: { /* filter tale words from kids json */ },
+//   };
+
+type KidsTranlsation = TranslationJSON & { image: string };
```

Comment:
This entire section from here to line 32 seems duplicate between the themes, no? Try to extract it into a common function and use it in both places :)